organisaatio-ui/src/components/Sivut/LomakeSivu/UusiToimija/PerustietoLomake/PerustietoLomake.tsx

```diff
@@ -152,13 +162,13 @@ export default function PerustietoLomake(props: OrganisaatioProps) {
                     />
                 </div>
                 <div className={styles.Kentta}>
-                    <label>{i18n.translate('PERUSTIETO_MUUT_KUNNAT')}</label>
+                    <label>{i18n.translate('PERUSTIETO_MUUT_KUNNATs')}</label>
                     <Select
                         isMulti
                         value={
-                            organisaatio.muutKotipaikatUris && organisaatio.muutKotipaikatUris.length
-                                ? kaikkiKunnat.filter(
-                                      (kk) => !!organisaatio.muutKotipaikatUris.find((mku: string) => mku === kk.value)
+                            organisaatio.muutKotipaikatUris
+                                ? organisaatio.muutKotipaikatUris.map(
```

Comment:
dunno but could the ternary be replaced with `(organisaatio.muutKotipaikatUris || [])`?

Reply:
yes, I actually like this